vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -151,6 +153,7 @@ func main() {
 	}()
 
 	if err = server.ListenAndServeTLS("", ""); err != nil {
-		klog.Fatalf("HTTPS Error: %s", err)
+		klog.ErrorS(err, "Failed to start server")
```

Comment:
```suggestion
		klog.ErrorS(err, "Failed to start HTTPS server")
```

The HTTPS part may be relevant to the error?

Reply:
Thanks! Added